README.md

```diff
@@ -167,6 +167,45 @@ The daemon uses the following configuration options from the configuration file:
   spool being the default. If spool is chosen, records will be spooled and sent
   at a later time.
 
+Data reported
+---------------------
+
+The data reported by the telemetry client could be understood as two main sets a
+metadata and a payload.
+
+The metadata is used to report details of a machine's architecture the
+following are the metadata values currently collected (Record Format Version 4):
+
+* record_format_version: version of the record, currently is 'Version 4'. This
+  value changes when a new metadata is added.
+* classification: this value is used to differenciate and classify data. A
+  classification value must have two slashes separating three alphanumeric strings
+  and the strings should describe categories from most general to most specific
+  from left to right. i.e. myprobe/journal/error, aprobe/journal/warn
```

Comment:
For consistency, I think we should document classification exactly the same as in the telemetrics-backend README, and update the example classifications accordingly.

```
The "classification" field is used to identify the type of record sent by a
specific client probe; classifications use the format DOMAIN/PROBE/REST, where
DOMAIN is the vendor of the probe, PROBE is the probe name, and REST is a
probe-defined field to classify what is contained in the payload.
```